### PR TITLE
display the dashboard list

### PR DESCRIPTION
**Pull request by Nexucis**
This PR proposed a first version to display the list of available dashboard.

image

Signed-off-by: Augustin Husson <husson.augustin@gmail.com>

### CODE REVIEW

**Comment by Nexucis** [comment]
I'm moving this PR as draft since I have some trouble with the navigation. Somethings strange happens when passing from the list to the `DashboardView` it doesn't look it used the correct data.

Same thing when moving from `ViewDashboard` to the root. It doesn't get the list of dashboard. Instead the data is already filled with the value coming from the `ViewDashboard`. Which is super weird for me since `ViewDashboardList` is a different view

**Comment by Nexucis** [comment]
I'm wondering if it's coming from the warning you can see when you are going on the page https://localhost:3000/projects/perses/dashboards/Benchmark?start=6h manually

image

**Comment by sjcobb** [comment]
> I'm wondering if it's coming from the warning

Just pushed a commit to this branch that fixes the console error you mentioned!

File: ui/app/src/views/ViewDashboard.tsx [hunk 1]
```diff
@@ -41,36 +41,39 @@ function ViewDashboard() {
...
-                <DashboardView dashboardResource={data} datasourceApi={datasourceApi} />;
-              </ErrorBoundary>
-            </PluginRegistry>
-          </ChartsThemeProvider>
-        </ErrorBoundary>
-      </Box>
-    );
-  }
-  return null;
+  const { data, isLoading } = useDashboard(projectName, dashboardName);
```
Comment (sjcobb) [inline ui/app/src/views/ViewDashboard.tsx:52]: Pushed a commit that adjusts this loading state and now you can navigate from the dashboard listing to an individual dashboard. 

Something is still off with useDashboard results though and routing generally (back button is broken when trying to go back to the dashboard listing), but I need to log off for the day

Comment (Nexucis) [reply ui/app/src/views/ViewDashboard.tsx:52]: Thank you very much, everything is working now !

For the back button actually it's working, it's just the dashboard that is adding automatically the `?start=6h` so when you click on the back button, you can see it's removing the `?start=6h`, which is then added back automatically by the dashboard.

Comment (sjcobb) [reply ui/app/src/views/ViewDashboard.tsx:52]: Ah okay gotcha, I can look into fixing this in follow up PR

**Review by sjcobb — APPROVED** [review]

**Pull request merged**